vignettes/adding-support-for-reporting.Rmd

```diff
@@ -12,9 +12,12 @@ vignette: >
 
 ## Introduction
 
-The `teal` package offers an integrated reporting feature utilizing the `teal.reporter` package. For a comprehensive explanation of the reporting functionality itself, please refer to the documentation therein.
+The `teal` package offers an integrated reporting feature utilizing the `teal.reporter` package.
+For a comprehensive explanation of the reporting functionality itself, please refer to the documentation therein.
 
-This article is intended for module developers and aims to provide guidance on enhancing a custom `teal` module with an automatic reporting feature. This enhancement enables users to incorporate snapshots of the module outputs into a report which can then be reviewed in another module automatically provided by `teal`. Thus the app user can interact with the report.
+This article is intended for module developers and aims to provide guidance on enhancing a custom `teal` module with an automatic reporting feature.
+This enhancement enables users to incorporate snapshots of the module outputs into a report which can then be reviewed in another module automatically provided by `teal`.
+Thus the app user can interact with the report.
```

Comment:
@chlebowa Do you intend to maintain this as a separate paragraph? if yes then this need addtional  line break. Currently, on the knitting RMD, it's not appearing as a distinct paragraph on the page.

<img width="648" alt="image" src="https://github.com/insightsengineering/teal/assets/6700955/287ee992-ae65-46b1-8889-a270d3735f57">

Reply:
I dont think he did, this is just so that each sentence is in a new line of rmd : P